extensions/scheduler/runtime/pom.xml

```diff
@@ -18,6 +18,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-arc</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This does not seem right. 

We now have the `quartz` extension as part of https://github.com/quarkusio/quarkus/pull/5481 

Can you rebase with master and do a full build. I am not seeing any errors.

Reply:
This was against latest master (commit `5dca04d5e45d19a67655b67f4fc1dbf84ca68edd`). I'll redo again and see if there's something more going on.

Reply:
Hello @machi1990, sorry about the noise. It looks like I had a class called `DelegatingThreadPool` locally (not committed) which was lying around from my previous attempt at experimenting with using a custom thread pool for quartz. 

This is what is now failing compilation. Sorry about the noise. I'll close this one.

Reply:
@jaikiran thanks for confirming :-)